### ISSUE TITLE
Truffle init not working 

### ISSUE THREAD

**Issue by cmaliwal** [trufflesuite/truffle#1589]
Getting error when I am running `truffle init` command .

```
chirag@chirag-Lenovo-ideapad-330-15IKB:~/projects/blockchain-work/BrtTokenSale$ truffle init
Downloading...
Error: Truffle Box at URL https://github.com/truffle-box/bare-box.git doesn't exist. If you believe this is an error, please contact Truffle support.
    at Request._callback (/usr/lib/node_modules/truffle/build/webpack:/packages/truffle-box/lib/utils/unbox.js:50:1)
    at Request.self.callback (/usr/lib/node_modules/truffle/build/webpack:/~/request/request.js:185:1)
    at Request.emit (events.js:182:13)
    at Request.<anonymous> (/usr/lib/node_modules/truffle/build/webpack:/~/request/request.js:1157:1)
    at Request.emit (events.js:182:13)
    at IncomingMessage.<anonymous> (/usr/lib/node_modules/truffle/build/webpack:/~/request/request.js:1079:1)
    at Object.onceWrapper (events.js:273:13)
    at IncomingMessage.emit (events.js:187:15)
    at endReadableNT (_stream_readable.js:1094:12)
    at process._tickCallback (internal/process/next_tick.js:63:19)
chirag@chirag-Lenovo-ideapad-330-15IKB:~/projects/blockchain-work/BrtTokenSale$ truffle init
Downloading...
Error: Truffle Box at URL https://github.com/truffle-box/bare-box.git doesn't exist. If you believe this is an error, please contact Truffle support.
    at Request._callback (/usr/lib/node_modules/truffle/build/webpack:/packages/truffle-box/lib/utils/unbox.js:50:1)
    at Request.self.callback (/usr/lib/node_modules/truffle/build/webpack:/~/request/request.js:185:1)
    at Request.emit (events.js:182:13)
    at Request.<anonymous> (/usr/lib/node_modules/truffle/build/webpack:/~/request/request.js:1157:1)
    at Request.emit (events.js:182:13)
    at IncomingMessage.<anonymous> (/usr/lib/node_modules/truffle/build/webpack:/~/request/request.js:1079:1)
    at Object.onceWrapper (events.js:273:13)
    at IncomingMessage.emit (events.js:187:15)
    at endReadableNT (_stream_readable.js:1094:12)
    at process._tickCallback (internal/process/next_tick.js:63:19)
chirag@chirag-Lenovo-ideapad-330-15IKB:~/projects/blockchain-work/BrtTokenSale$ truffle version
Truffle v4.1.14 (core: 4.1.14)
Solidity v0.4.24 (solc-js)
chirag@chirag-Lenovo-ideapad-330-15IKB:~/projects/blockchain-work/BrtTokenSale$ 
```

**Comment by ElShapovalov** [comment]
I have same problem!

**Comment by gnidan** [comment]
Truffle v5 broke this on v4. We released a maintenance build **v4.1.15** to fix this problem.

We recommend upgrading to v5 for new projects, or you can install the latest v4 release to get around this project in the meantime.

Thank you!
